src/ophyd_async/fastcs/panda/_block.py

```diff
@@ -32,26 +35,28 @@ class PulseBlock(Device):
     width: SignalRW[float]
 
 
-class PcompDirectionOptions(str, Enum):
+class PcompDirection(StrictEnum):
     positive = "Positive"
     negative = "Negative"
     either = "Either"
 
 
-EnableDisableOptions = SubsetEnum["ZERO", "ONE"]
+class BitMux(SubsetEnum):
+    zero = "ZERO"
+    one = "ONE"
```

Comment:
```suggestion
    ZERO = "ZERO"
    ONE = "ONE"
```

Reply:
Will do this in https://github.com/bluesky/ophyd-async/issues/605